cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -0,0 +1,645 @@
+@import 'cms/theme/variables-v1';
+@import 'elements/course-unit-mfe-iframe';
+
+.wrapper-content.wrapper {
+  padding: 5px;
+}
+
+body .wrapper-xblock {
+  background-color: $transparent;
+  border-radius: 6px;
+  border: none;
+
+  &:hover {
+    border-color: none;
+  }
+
+  .xblock-header-primary {
+    padding: 24px;
+    padding-bottom: 12px;
```

Comment:
[nit] Should be simplified to one `padding` rule.
Also, we can stick to general rules for margin/paddings with using `$baseline` variable (ex: https://github.com/openedx/edx-platform/blob/2bbd8ecd180fd299de2c4ee4bc1542ccb97b50bb/cms/static/sass/_base.scss#L313)

Reply:
Added